client/game.cpp

```diff
@@ -0,0 +1,138 @@
+#include "game.hpp"
+
+#include <devices/tda.hpp>
+
+#include "controllers/fps-controller.hpp"
+#include "server-message.hpp"
+#include "events.hpp"
+#include "event-system.hpp"
+#include "graphics/view.hpp"
+#include "graphics/texture-object.hpp"
+#include "graphics/material.hpp"
+#include "graphics/vertex-buffer-object.hpp"
+#include "graphics/renderable.hpp"
+#include "resources/pixel-buffer.hpp"
+
+namespace tec {
+	using networking::ServerMessage;
+	using networking::MessageType;
+
+	Game::Game(const unsigned int viewport_width, const unsigned int viewport_height) :
+		ps(this->simulation.GetPhysicsSystem()),
+		vcs(this->simulation.GetVComputerSystem()),
+		sound_thread([this] () { ss.Update(); }) {
```

Comment:
I think we should put the opening brace on its own line to clearly separate the body from the initialisation list.

Reply:
We can, in this case it seemed cleaner being a 1 liner.

Reply:
I mean the opening brace of the `Game` constructor, not of the lambda that `sound_thread` is initialised with.

Reply:
I mean the opening brace of the `Game` constructor, which isn't a 1 liner.

Reply:
I see it now, my bad on that one.